docs/release-notes/release-notes-0.16.0.md

```diff
@@ -0,0 +1,10 @@
+# Release Notes
+
+## RPC Server
+
+* [Add `min_creation_date` and `max_creation_date` filter fields to
+  `ListInvoiceRequest`](https://github.com/lightningnetwork/lnd/pull/6542)
```

Comment:
Release notes seem to be switched around for the two PRs :) Should be 6544 here + about payments.